### ISSUE TITLE
model.predict() should not require presence of "target"

### ISSUE THREAD

**Issue by pchalasani** [jdb78/pytorch-forecasting#960]
When I try to do `model.predict(df)` and `df` does not contain the target, I get an error.
In predict mode, it shouldn't care if the "target" is present or not, so it's unclear if I'm doing something wrong, or if it's expected behavior.

**Comment by foundationequities** [comment]
Can you post your training = TimeSeriesDataSet( ) statement. Where did you include "target"?

**Comment by daniwxcode** [comment]
I running into the same issue.

**Comment by daniwxcode** [comment]
> When I try to do `model.predict(df)` and `df` does not contain the target, I get an error. In predict mode, it shouldn't care if the "target" is present or not, so it's unclear if I'm doing something wrong, or if it's expected behavior.


Have you find a solution?

**Comment by pchalasani** [comment]
> > When I try to do `model.predict(df)` and `df` does not contain the target, I get an error. In predict mode, it shouldn't care if the "target" is present or not, so it's unclear if I'm doing something wrong, or if it's expected behavior.
> 
> Have you find a solution?

I have switched to using the DARTS package which is very similar and seems more intuitive, and more responsive devs.
https://github.com/unit8co/darts

**Comment by pchalasani** [comment]
Yes of course there is a work around where you insert a dummy target.

But I actually started using the DARTS package which seems more intuitive and the devs are more responsive.
